src/library/scala/concurrent/duration/Deadline.scala

```diff
@@ -64,7 +64,7 @@ case class Deadline private (time: FiniteDuration) extends Ordered[Deadline] {
   /**
    * The natural ordering for deadline is determined by the natural order of the underlying (finite) duration.
    */
-  def compare(other: Deadline) = time compare other.time
+  def compare(other: Deadline): Int = time compare other.time
```

Comment:
IMO the more helpful change is to add optional `override`. As in other cases, the return type is constrained.

Reply:
@som-snytt Thanks, for the overriding thing, that's true, but would that will be more place to add the override?

Reply:
I should have said `IMHO` before. Not a big deal, but I would not add the explicit type unless it added information. In this case, the type has to conform because its an override. Oh, it's an override? The optional override here is helpful to me because it comes after a bunch of other definitions. But there is no policy on this codebase, so this is just my opinion. Some people don't like override for implementing method.

Reply:
Thanks, @som-snytt ,I'm ok to revert this kind back, I added them for consistency too, other's methods there has an explicit return type.